lib/common/src/main/java/dk/alexandra/fresco/lib/common/collections/Collections.java

```diff
@@ -1,17 +1,28 @@
-package dk.alexandra.fresco.framework.builder.numeric;
+package dk.alexandra.fresco.lib.common.collections;
 
 import dk.alexandra.fresco.framework.DRes;
 import dk.alexandra.fresco.framework.builder.ComputationDirectory;
+import dk.alexandra.fresco.framework.builder.numeric.ProtocolBuilderNumeric;
 import dk.alexandra.fresco.framework.util.Pair;
-import dk.alexandra.fresco.framework.util.RowPairD;
 import dk.alexandra.fresco.framework.value.SInt;
-import dk.alexandra.fresco.lib.collections.Matrix;
+import dk.alexandra.fresco.lib.common.math.DefaultAdvancedNumeric;
+import dk.alexandra.fresco.lib.common.util.RowPairD;
 import java.math.BigInteger;
 import java.util.List;
 
 /** Interface for operations on collections. This also includes opening and closing values. */
 public interface Collections extends ComputationDirectory {
```

Comment:
I realize this is not a part of the merge, but the name is unfortunate.

Maybe call it MatrixCol or similar, Collections reside already in java....